CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-## [v0.5.4] - 2020-08-26
+## [v0.5.5] - 2020-08-26
```

Comment:
I just noticed the date is wrong too, should be 2020-08-27

Also, please add so: "....use **of** the `cortex_m`....